ibm_catalog.json

```diff
@@ -0,0 +1,145 @@
+                {
+                  "title": "Create Virtual Private Cloud (VPC)",
+                  "description": "No"
+                },
+                {
+                  "title": "Create Red Hat OpenShift Cluster",
+                  "description": "Yes"
+                },
+                {
+                  "title": "Use Existing VPC instance",
+                  "description": "No"
+                },
+                {
+                  "title": "New Resource Group Creation",
+                  "description": "No"
+                }
+              ],
+              "diagrams": [
+                {
+                  "diagram": {
+                    "caption": "Red Hat OpenShift cluster with AI addon for IBM Cloud.",
+                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ocp-ai/blob/main/reference-architecture/deployable-architecture-ocp-cluster-qs.svg",
+                    "type": "image/svg+xml"
+                  },
+                  "description": "This architecture illustrates the deployment of a single-zone Red Hat OpenShift cluster integrated with an AI addon, showcasing the key components and their interactions for AI workload support."
+                }
+              ]
+            },
+            "configuration": [
+              {
+                "key": "default_worker_pool_machine_type",
+                "description": "Specifies the machine type for the default worker pool. This determines the CPU, memory, and disk resources available to each worker node. Choose a machine type that is suitable for your AI workload's computational requirements. Refer [IBM Cloud documentation for available machine types](https://cloud.ibm.com/docs/openshift?topic=openshift-vpc-flavors)"
+              },
+              {
+                "key": "default_worker_pool_workers_per_zone",
+                "description": "Defines the number of worker nodes to provision in each zone for the default worker pool. The cluster must have at least 2 worker nodes. Each worker node must have 8vCPU and 32GB memory and a GPU. [Learn More](https://cloud.ibm.com/docs/openshift?topic=openshift-ai-addon-install&interface=ui#ai-min)"
+              },
+              {
+                "key": "default_worker_pool_operating_system",
+                "description": "Selects the operating system for the worker nodes in the default worker pool. Ensure that the selected operating system is compatible with your AI framework and dependencies. Refer [here](https://cloud.ibm.com/docs/openshift?topic=openshift-openshift_versions) for supported Operating Systems"
+              },
+              {
+                "key": "additional_worker_pools",
+                "description": "Define additional worker pools with custom configurations (e.g., different machine types or operating systems) to accommodate diverse AI workload requirements within the cluster, such as GPU-optimized pools for deep learning tasks."
```

Comment:
these descriptions should be in varaibles.tf of ocp.ai DA

Reply:
Done